client/src/containers/approver/proposals/Rejected.js

```diff
@@ -99,15 +99,22 @@ class Rejected extends Component {
     let diff2 = users && rejectedProposals.filter(
       proposal => !users.find(user => user.id === proposal.opener)
     );
+    let diff3 = users && rejectedProposals.filter(
+      proposal => !users.find(user => user.id === proposal.closer)
+    );
     diff = roles ?
       diff.map(proposal => proposal.object) :
       rejectedProposals.map(proposal => proposal.object);
     diff2 = users ?
       diff2.map(proposal => proposal.opener) :
       rejectedProposals.map(proposal => proposal.opener);
+    diff3 = users ?
+      diff2.map(proposal => proposal.closer) :
```

Comment:
Rejected screen incorrectly displays the avatar with a question mark glyph. Change `diff2` → `diff3`. Line 112.

Reply:
See below comment for resolution.